scripts/build-package.js

```diff
@@ -0,0 +1,36 @@
+const glob = require('glob');
+const path = require('path');
+const npmRun = require('npm-run');
+const yargs = require('yargs');
+
+/**
+ * Runs the build-babel or build-webpack npm target for the specified package.
+ * @param packageName Name of the package as listed in ./packages/
+ */
+function runBuildTask(packageName) {
+  const npmTask = packageName !== 'react-component-playground' ? 'build-babel' :
+      'build-webpack';
+  const stdout = npmRun.execSync(`PACKAGE=${packageName} npm run ${npmTask}`, {
+    cwd: __dirname,
+  });
+
+  /* eslint-disable no-console */
+  console.log(stdout.toString('utf-8'));
+}
+
+glob('./packages/*react-*', null, (err, files) => {
+  const allPackages = files.map(f => path.basename(f));
+  const argv = yargs
+      .usage('Usage: $0 <package>')
+      .help()
+      .argv;
+  const targetPackage = argv._[0];
+
+  if (!targetPackage) {
+    allPackages.forEach(pkg => (runBuildTask(pkg)));
```

Comment:
This needs to be a bit smarter. I.e. packages bundled via webpack need to be built after other ones they depend on.

![image](https://cloud.githubusercontent.com/assets/250750/21639270/d671e1c6-d277-11e6-8584-578da412a2a3.png)


Suggestion:
1. Keep a list of packages bundled with webpack (just CP for now)
2. Run packages that aren't included in that list before those that are
3. Bonus: Use same list to check which task to run (line 11)

Accepted limitation: This might still fail when more builds that bundle with webpack depend on each other. There's only one webpack build now (CP) and there are no plans for any other in the near/mid future.